po/meson.build

```
@@ -2,7 +2,7 @@ i18n.gettext(meson.project_name(),
     args: [
         '--directory=' + meson.source_root(),
         '--from-code=UTF-8',
-        '-cTRANSRATORS'
+        '-cTRANSLATORS'
```

```suggestion
        '-cTRANSLATORS:'
```

This should resolve the issue :)